nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -0,0 +1,49 @@
+collision_monitor:
+  ros__parameters:
+    use_sim_time: True
+    base_frame_id: "base_footprint"
+    odom_topic: "/odom"
+    cmd_vel_in_topic: "cmd_vel_raw"
+    cmd_vel_out_topic: "cmd_vel"
+    transform_tolerance: 0.5
+    data_timeout: 5.0
+    # Polygons represent safety area around the robot for "stop" and "slowdown" action types
+    polygons: ["PolygonStop"]
+    PolygonStop:
+      type: "polygon"
+      points: [0.5, 0.5, 0.5, -0.5, -0.1, -0.5, -0.1, 0.5, 0.5, 0.5]
+      action_type: "stop"
+      max_points: 3
+      visualize: True
+      polygon_topic: "polygon_stop"
+    PolygonSlow:
+      type: "polygon"
+      points: [0.5, 0.5, 0.5, -0.5, -0.1, -0.5, -0.1, 0.5, 0.5, 0.5]
+      action_type: "slowdown"
+      max_points: 3
+      slowdown_ratio: 0.3
+      visualize: True
+      polygon_topic: "polygon_slowdown"
+    # Approaching to obstacles with constant time before a collision
+    approach: False
+    footprint_name: "FootprintApproach"
```

Comment:
Make these under the namespace of the  specific polygon. Then you don't need `footprint_name` parameter at all and its makes way more sense for the time before collision / time step to be there too. 

I don't understand the `approach: false` in L28

Reply:
In the latest update all footprint parameters were moved inside `Polygon` and its derived `Circle`.